public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
- [ ] It would be better to add a unit testing for the application as mentioned in the previous comments